suites/quincy/cephfs/tier-1_cephfs_mirror.yaml

```diff
@@ -0,0 +1,254 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_cephfs_mirror.yaml
+# Test-Case: Configure CephFS Mirror setup and run IOs
```

Comment:
Do we also need mention about relevant/suggested conf file for the test suites in documentation?